File: programmers/lv1/3진법뒤집기.py
def solution(n):
    answer = 0
    triple = []
    while(n > 0):
        triple.append(n % 3)
        n = n // 3
    triple.reverse()
    for i in range(len(triple)):
        answer += triple[i] * (3**i)
    return answer
solution(125)

File: programmers/lv1/3진법뒤집기.js
function solution(n) {
  var answer = [];

  while (n > 1) {
    answer.push(n % 3);
    n = Math.floor(n / 3);
    console.log(n);
  }

  if (n > 0) answer.push(n);

  let cnt = 1;
  let hap = 0;
  while (answer.length > 0) {
    hap += cnt * answer.pop();
    cnt *= 3;
  }
  return hap;
}
